ui/src/app/views/workspace-edit/component.html

```diff
@@ -1,53 +1,80 @@
 <div *ngIf="!notFound else notFoundMessage">
-  <ng-container [ngSwitch]="mode">
-    <div *ngSwitchCase="Mode.Create">
-      <h2>Create a new Workspace</h2>
-    </div>
-    <div *ngSwitchCase="Mode.Edit">
-      <h2>Edit workspace "{{workspace.name}}"</h2>
+  <div class="section name-section">
+    <div class="required-field">
+      <ng-container [ngSwitch]="mode">
+        <div *ngSwitchCase="Mode.Create">
+          <h2 class="page-header">Create a new Workspace</h2>
+        </div>
+        <div *ngSwitchCase="Mode.Edit">
+          <h2 class="page-header">Edit workspace "{{workspace.name}}"</h2>
+        </div>
+        <div *ngSwitchCase="Mode.Clone">
+          <h2 class="page-header">Clone workspace "{{oldWorkspaceName}}"</h2>
+        </div>
+      </ng-container>
+      <div class="required-text">(Required)</div>
     </div>
-    <div *ngSwitchCase="Mode.Clone">
-      <h2>Clone workspace "{{oldWorkspaceName}}"</h2>
+    <div class="name-area required-field" [class.validation-error]="nameNotEntered">
+      <input type="text" class="input name" name="workspace-name" autofocus
+             [(ngModel)]="workspace.name" [disabled]="!hasPermission"
+             placeholder="Workspace Name">
     </div>
-  </ng-container>
-  <div>* is a required field</div>
+  </div>
   <div>
-    <div class="name-area" [class.validation-error]="nameNotEntered">
-      <input type="text" class="input name" name="workspace-name" autofocus
-          [(ngModel)]="workspace.name" [disabled]="!hasPermission"
-          placeholder="Workspace Name">
-      <span style="display: inline-block; vertical-align: bottom">
-        <div>Billing Account:</div>
-        <div>National Institute of Health</div>
-      </span>
+
+    <div class="section billing-project-section">
+      <div class="section-header">
+        Billing Account
+      </div>
+      <div class="section-sub-header">
+        National Institutes of Health
+      </div>
+      <div class="section-text">
+        To fulfill program requirements, All of Us requests the following information
+        for each workspace.<!-- If you are not ready to complete this questionnaire, or if-->
+        <!--this is a temporary workspace, please check this box (any entries already made-->
+        <!--will not be saved):-->
+      </div>
+      <!--<clr-checkbox-->
+          <!--name="not-ready"-->
+          <!--id="not-ready"-->
+          <!--[(ngModel)]="workspace.researchPurpose.notReady"-->
+          <!--[clrDisabled]="mode == Mode.Edit">-->
+        <!--<h3>Not Ready</h3>-->
+      <!--</clr-checkbox>-->
```

Comment:
I'm trying to figure out what this PR is intending to cover and it seems like the requirements have not been finalized yet. The commented out code reflects currently un-accepted edits in the working google doc, not actual requirements. This should probably be removed until requirements are finalized. Even better, let's finalize requirements and then add the code.

Reply:
I don't know why they are unaccepted edits. I will follow up on that. I have checked with Fritz and that is finalized, and has been moved into mocks. See this mock: https://projects.invisionapp.com/d/#/console/12820961/275889951/preview